containers/shop/balance.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <div class="balance-tooltip">
+    You have 1337
```

Comment:
How about this number is moved to the store and a getter is used instead?

Reply:
Done